Cesium.CodeGen/Ir/Expressions/ConstantExpression.cs

```diff
@@ -34,6 +34,7 @@ private static IConstant GetConstant(Ast.ConstantExpression expression)
             CTokenType.IntLiteral => new IntegerConstant(constant.Text),
             CTokenType.CharLiteral => new CharConstant(constant.Text),
             CTokenType.StringLiteral => new StringConstant(constant),
+            CTokenType.FloatLiteral => new FloatConstant(constant.Text), //TODO: Add DoubleConstant
```

Comment:
As per C11 clause 6.4.4.2, “An unsuffixed floating constant has type double. If suffixed by the letter f or F, it has type float. If suffixed by the letter l or L, it has type long double.“

So I think we can distinguish between `float`, `double` and `long double` trivially.

Reply:
In current implementation we don't have suffix support. I have added a todo comment to get back to it when it is supported

Reply:
Then probably would be good to have Double by default, at least that would be correct C behaviour, and if we miss `float` support, because of `f` suffix missing, then let's add `float` support later one.
Regarding missing suffix support, is this on us or on Yoakke?

Reply:
Suffix support is on us, because I want us to have a separate lexer eventually. Not sure if Yoakke already supports the suffices.